Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/adapters/GameAdapter.java

```diff
@@ -222,17 +224,27 @@ public boolean onLongClick(View view)
 		GameViewHolder holder = (GameViewHolder) view.getTag();
 
 		// Get the ID of the game we want to look at.
-		// TODO This should be all we need to pass in, eventually.
-		// String gameId = (String) holder.gameId;
+		String gameId = (String) holder.gameId;
 
 		FragmentActivity activity = (FragmentActivity) view.getContext();
-		GameDetailsDialog.newInstance(holder.title,
-				holder.description,
-				holder.country,
-				holder.company,
-				holder.path,
-				holder.screenshotPath).show(activity.getSupportFragmentManager(), "game_details");
 
+
+		AlertDialog.Builder builder = new AlertDialog.Builder(activity);
```

Comment:
The same logic need to be applied for the AndroidTV UI inside ```GameRowPresenter.onBindViewHolder``` using ```holder.cardParent.setOnLongClickListener(showPerGameSettingsDialog())```

Reply:
@mahdihijazi I don't have the capacity to test Android TV, but I did implement this now